CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### v0.36.2
+###### CloudFormation Specifications
+- Update CloudFormation specs to 18.5.0 (pull #[1715](https://github.com/aws-cloudformation/cfn-python-lint/pull/1715))
+- Get `AllowedValues` from Botocore (pull #[1682](https://github.com/aws-cloudformation/cfn-python-lint/pull/1682))
```

Comment:
```suggestion
- Get `AllowedValues` from Botocore during `--update-specs` (pull #[1682](https://github.com/aws-cloudformation/cfn-python-lint/pull/1682))
```
think this change is fundamental enough to at least bump minor version to `0.37.0` as well

Reply:
Should https://github.com/aws-cloudformation/cfn-python-lint/pull/1716 be included in the changelog as well?

Reply:
Yes.  I'll add it